Content/contributing corrections (#513)
centrica-engineering/nucleus-docs: src/content/docs/guidelines/contributing.mdx

Everyone.

Individuals from various professional backgrounds, such as Designers, Software Engineers, Testers, Solutions Architects, Product Owners, and Directors, are all welcome to contribute to Nucleus.

## What to expect

Everyone involved in the process plays a significant role in creating a customer experience that is coherent, consistent, and engaging.

A contribution flow has been established by the community to ensure that any contribution made adheres to the standards set by Nucleus.

### Have you identified a problem?

If you have, please share the problem with the community. Asking if anyone else is aware of the same or similar problem.

You'll be surprised how many problems exist that are constantly overlooked and ignored, so any problem, no matter how big or small has the potential to add value.

If a solution will be...

- reusable by another team/journey
- a solution to a new problem or an extension of an existing solution

> …the chances are the idea would be a great contribution!

Reviewer: I'm not keen on ellipsis, will this sentence work without it?
```suggestion
The chances are the idea would be a great contribution!
```

Author: Rephrased the whole paragraph to work without ellipsis.

> The chart below helps when making decisions about what to do:

Reviewer: ```suggestion
This chart will help when making decisions about what to do:
```
